### ISSUE TITLE
volcanosv-vc-large-indel.py fails

### ISSUE THREAD

**Issue by DayTimeMouse** [maiziezhoulab/VolcanoSV#9]
Hi,

I ran `volcanosv-vc-large-indel.py`, but the results of chr* did not have volcano_variant_chr*vcf file.

log:
```
Traceback (most recent call last):
  File "/home/xgc/variant/sv/volcanosv/VolcanoSV/bin/VolcanoSV-vc/Large_INDEL//extract_contig_signature_Hifi.py", line 740, in <module>
    ref_seq = dc_ref[chr_name]
KeyError: 'chr1'
cat: large_indel_output_tumor/chr1/volcano_variant_chr*vcf: 没有那个文件或目录
```
script:
```
path_to_volcanosv='../VolcanoSV'
for i in {1..22}
do
echo "***********************large-indel for chr${i}*******************"
python3 ${path_to_volcanosv}/bin/VolcanoSV-vc/Large_INDEL/volcanosv-vc-large-indel.py \
-i volcanosv_asm_output_tumor \
-bam hifi_tumor.bam \
-o large_indel_output_tumor \
-ref genome.fa \
-t 20 \
-chr ${i} \
-dtype Hifi \
-px hifi_tumor
done
```

Best regards.

**Comment by volcano1998** [comment]
What is your reference file like? Does the contig have name in a format like chr1 chr2 etc?

**Comment by DayTimeMouse** [comment]
Yes, genome is T2T-CHM13, the format is chr1, chr2...

**Comment by volcano1998** [comment]
Your error is a new one to me. Is it possible to send me a small BAM file (like chr22) and the contig for that chromosome as well? So I can reproduce the error in my end and try to debug.

**Comment by volcano1998** [comment]
Does your reference file have some descriptive fields after the chromosome name? For example
`>chr1          xxxxxxx`
format like that.

**Comment by DayTimeMouse** [comment]
Yes, after I simplied chr id, `volcanosv-vc-large-indel.py` works well.
But `ValueError: too many values to unpack (expected 2) in volcanosv-vc-small-indel.py` is still exist.

The small BAM file (like chr22) size is huge, ~1.16G, how can I send to you?

**Comment by volcano1998** [comment]
Is it possible to upload to a cloud drive (google drive or anything similar) and share the link to me please?

I need the BAM, contig and reference file to test. Thanks!

**Comment by DayTimeMouse** [comment]
Hi, I have uploaded bam file and contig to google drive(https://drive.google.com/drive/folders/1w6pn5bIiiNIWmafqqLCwgqod06h3jZjM?usp=drive_link).

Thanks in advance.

**Comment by volcano1998** [comment]
Thanks! I also need the assembled contig file generated by VolcanoSV-asm, which should be under volcanosv-asm_output/chr22/final_contigs/<prefix>_final_contigs.fa. Could you also provide that please?

**Comment by DayTimeMouse** [comment]
Hi,

I uploaded the contig file now in the same link. Please have a check.

**Comment by volcano1998** [comment]
Hi @DayTimeMouse I was able to generate a VCF with ~1600 variants. I had to manually modify some scripts since you gave me a single chromosome reference file.

I think the problem might be the reference splitting. Can you check if you have a file in large_indel_output_tumor/chr1/ref_by_chr/chr1.fa ? And how big it is ?

**Comment by DayTimeMouse** [comment]
Hi [volcano1998](https://github.com/volcano1998),

I'm so sorry for that. Now I uploaded the full reference file (https://drive.google.com/drive/folders/1w6pn5bIiiNIWmafqqLCwgqod06h3jZjM?usp=drive_link).

large_indel_output_tumor/ref_by_chr/chr1.fa size is 239.74 MB, and large_indel_output_tumor/ref_by_chr/chr22.fa size is 49.56MB.

**Comment by volcano1998** [comment]
Thank you! I used your wgs reference file and was able to get the VCF without changing my code. I ran it on chr22 and got a VCF of size 3.1M in 3 mins.

I also notice that your file structure is the older version of VolcanoSV. I suggest you to try git pull and run it on chr22 again.

Here is my command.

```
python3 ${path_to_volcanosv}/bin/VolcanoSV-vc/Large_INDEL/volcanosv-vc-large-indel.py \
	-i ../volcanosv_asm_output/ \
	-o volcanosv_large_indel_output/ \
	-dtype Hifi \
	-bam ../chr22.bam \
	-ref ../chm13.id.simply.fa  \
	-chr 22 -t 10 \
	-px hifi_tumor
```
